test/lit/passes/optimize-instructions-ignore-traps.wast

```diff
@@ -1,5 +1,5 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
-;; RUN: wasm-opt %s --all-features --optimize-level=2 --ignore-implicit-traps -S -o - \
+;; RUN: wasm-opt %s --all-features --optimize-instructions --optimize-level=2 --ignore-implicit-traps -S -o - \
```

Comment:
Can you add these tests in a new file to avoid making unrelated changes to other tests in this file?

Reply:
Basically this lit required this `--optimize-instructions`. It was miss during porting I guess

Reply:
Oh, I see. It's concerning that we accidentally dropped the key option for the test. I'll take a look at what went wrong there.